source/StartConditions.h

```diff
@@ -54,6 +54,11 @@ class StartConditions : public CoreStartData {
 	const Sprite *GetThumbnail() const noexcept;
 	const std::string &GetDisplayName() const noexcept;
 	const std::string &GetDescription() const noexcept;
+	const std::string &GetHint() const noexcept;
+
+	bool Visible(ConditionsStore conditionsStore) const;
+	bool Revealed(ConditionsStore conditionsStore) const;
+	bool Unlocked(ConditionsStore conditionsStore) const;
```

Comment:
Same needs done for the .cpp file.
```suggestion
	bool Visible(const ConditionsStore &conditionsStore) const;
	bool Revealed(const ConditionsStore &conditionsStore) const;
	bool Unlocked(const ConditionsStore &conditionsStore) const;
```